docs/sources/operators-guide/configuring/configuring-custom-trackers.md

```diff
@@ -25,14 +25,37 @@ Each custom tracker counts the active series matching its label pattern on a per
 
 Series with metric name `cortex_ingester_active_series_custom_tracker` have two labels applied: `name` and `user`. The value of the `name` label is the name of the custom tracker specified in the configuration. The value of the `user` label is the tenant-id for which the series count applies.
 
-Assume two custom trackers are configured as in the example above, and that your Grafana Mimir cluster has three tenants: `tenant_1`, `tenant_2`, and `tenant_with_only_prod_metrics`. Assume all series within `tenant_with_only_prod_metrics` have labels that match the pattern `{namespace=~"prod-.*"}` and none that match `{namespace=~"dev-.*"}`.
+To illustrate, assume that two custom trackers are configured as in the yaml shown above, and that your Grafana Mimir cluster has two tenants: `tenant_1` and `tenant_with_only_prod_metrics`. Assume that `tenant_with_only_prod_metrics` has 3 series with labels that match the pattern `{namespace=~"prod-.*"}` and none that match `{namespace=~"dev-.*"}` and that `tenant_1` has 5 series that match the pattern `{namespace=~"dev-.*"}` and 10 series that match the pattern `{namespace=~"prod-.*"}`.
 
-In this example, the following output appears when the `/metrics` endpoint for the ingester component is scraped:
+You should see the following output when the `/metrics` endpoint for the ingester component is scraped:
```

Comment:
```suggestion
In this example, the following output appears when the `/metrics` endpoint for the ingester component is scraped:
```
Revert this because:
* The term _should_ is ambiguous. Rephrase it so that the advice is unambigous and accurate.